Woof/Woof/View/OwnerFlow/OwnerProfileView/EditOwnerInformationView.swift

```diff
@@ -24,10 +24,10 @@ struct EditOwnerInformationView: View {
             TextField("Enter your surname", text: $surname)
             TextField("Enter your phone", text: $phone)
             TextField("Enter your city", text: $city)
```

Comment:
Does it make sense to mark the * the mandatory fields and update the UI as well to more readability
```suggestion
            TextField("Enter your surname *", text: $surname)
            TextField("Enter your phone *", text: $phone)
           ....
```

``` swift
Text("Fields with * are mandatory")
```

Reply:
Added with e8e2fd4